src/assets/i18n/nl.json

```diff
@@ -1,1278 +1,1265 @@
+  "When set, the following text will be shown prior to showing login page to the user": "Indien ingesteld, wordt de volgende tekst weergegeven voordat de inlogpagina aan de gebruiker wordt getoond",
+  "Widget Category": "Widget categorie",
+  "Widget Editor": "Widget editor",
+  "Widget Subtext": "Widget subtekst",
+  "Widget Text": "Widget tekst",
+  "Widget Title": "Widget titel",
+  "Widget Type": "Widget type",
+  "Widget has errors": "Widget heeft fouten",
+  "Widget {slot} Settings": "Widget {slot} instellingen",
+  "Widgets": "Widgets",
+  "Winbind NSS Info": "Winbind NSS-informatie",
+  "Xen: Extent block size 512b, TPC enabled, Xen compat mode enabled, SSD speed": "Xen: Omvang blokgrootte 512b, TPC ingeschakeld, Xen-compatibiliteitsmodus ingeschakeld, SSD-snelheid",
+  "You can search both for local groups as well as groups from Active Directory. Press ENTER to separate entries.": "Je kunt zowel zoeken naar lokale groepen als naar groepen uit Active Directory. Druk op ENTER om de items te scheiden.",
+  "Your dashboard is currently empty!": "Je dashboard is momenteel leeg",
+  "ZFS Errors": "ZFS fouten",
+  "ZFS Replication to another TrueNAS": "ZFS replicatie naar een andere TrueNAS",
+  "ZFS Reports": "ZFS-rapportages",
+  "ZFS Stats": "ZFS-statistieken",
+  "ZFS Utilization": "ZFS-gebruik",
+  "Zoom In": "Inzoomen",
+  "Zoom Out": "Uitzoomen",
+  "Zvol Location": "Zvol locatie",
+  "Zvol Space Management": "Zvol ruimtebeheer",
+  "by ancestor": "per voorouder",
+  "details": "details",
+  "disk stats": "schijfstatistieken",
+  "disk writes": "schijf schrijft",
+  "expires in {n, plural, one {# day} other {# days} }": "verloopt over {n, plural, one {# dag} other {# dagen} }",
+  "iSCSI Extent": "iSCSI Extent",
+  "iSCSI Initiator": "iSCSI Initiator",
+  "iSCSI Share": "iSCSI Share",
+  "iSCSI Target": "iSCSI Target",
+  "on this enclosure.": "op deze behuizing.",
+  "or": "of",
+  "pbkdf2iters": "pbkdf2iters",
+  "standby": "stand-by",
+  "to another TrueNAS": "naar andere TrueNAS",
+  "to cloud": "naar cloud",
+  "{ n, plural, one {# snapshot} other {# snapshots} }": "{ n, plural, one {# snapshot} other {# snapshots} }",
+  "{bits}/s": "{bits}/s",
+  "{comparator} (Range In)": "{comparator} (bereik binnen)",
+  "{comparator} (Range Not In)": "{comparator} (bereik niet binnen)",
+  "{coreCount, plural, one {# core} other {# cores} }": "{coreCount, plural, one {# core} other {# cores} }",
+  "{email} via {server}": "{email} via {server}",
+  "{failedCount} of {allCount, plural, =1 {# task} other {# tasks}} failed": "{failedCount} van {allCount, plural, =1 {# taak is} other {# taken zijn}} mislukt",
+  "{license} contract, expires {date}": "{license} contract, verloopt op {date}",
+  "{n, plural, =0 {No errors} one {# Error} other {# Errors}}": "{n, meervoud, =0 {Geen fouten} one {# Fout} other {# fouten}}",
```

Comment:
`meervoud` should remain `plural` 

This issue breaks validation.

Right version: 
```
"{n, plural, =0 {No errors} one {# Error} other {# Errors}}": "{n, plural, =0 {Geen fouten} one {# Fout} other {# fouten}}",
```